application/helpers/mpdf_helper.php

```diff
@@ -75,11 +75,17 @@ function pdf_create(
     }
 
     // Set the footer if voucher is invoice and if set in settings
-    if (!empty($CI->mdl_settings->settings['pdf_invoice_footer'])) {
+    if (!empty($CI->mdl_settings->settings['pdf_invoice_footer']) && $isInvoice) {
```

Comment:
Little explanation about this, I used $isInvoice because if I use 
`strpos($filename, trans('invoice')) !== false`

The footer will be also included in the report invoice_aging

`trans('invoice_aging')`

(as it seeks for invoice in the title of the file ...) 

If it's really a problem feel free to tell me and I will commit with `trans('invoice')` instead of `$isInvoice`